web-client/src/presenter/computeds/trialSessionWorkingCopyHelper.ts

```diff
@@ -101,12 +102,21 @@ export const trialSessionWorkingCopyHelper = (
       key: 'statusUnassigned',
       label: 'Unassigned',
     });
-
+  const trialStatusCounts = trialSession.calendaredCases.reduce((counters, c) => {
+    if(caseMetadata[c.docketNumber] === undefined || caseMetadata[c.docketNumber].trialStatus === "" 
+      || caseMetadata[c.docketNumber].trialStatus === undefined) {
+      counters["statusUnassigned"] = (counters["statusUnassigned"] || 0) + 1;
+    } else {
+      counters[caseMetadata[c.docketNumber].trialStatus] = (counters[caseMetadata[c.docketNumber].trialStatus] || 0) + 1;
+    }
+    return counters
+  }, {});
```

Comment:
Nice work @alexdziarn--this is a _much_ better approach. The disjunctive statement (x || y || z) can probably be improved, maybe with a self-documenting helper function like `trialStatusIsUnassigned`, but otherwise I think this is getting close. I'll take a look again tomorrow :)